README.adoc

```diff
@@ -1,9 +1,8 @@
-= Audio Library for Arduino =
-
-The Audio library enables an Arduino Zero or MKR1000 board to play back .wav files from a storage device like an SD card.
+= AudioZero Library for Arduino =
 
+The AudioZero library enables an Arduino SAMD Board (e.g., [MKR boards](https://store.arduino.cc/catalogsearch/result/?q=mkr), [Nano 33 IoT](https://store.arduino.cc/arduino-nano-33-iot), [Zero](https://store.arduino.cc/arduino-zero)) to play back .wav files from a storage device like an SD card to the `DAC0`/`A0` pin.
```

Comment:
```suggestion
The AudioZero library enables an Arduino SAMD Board (e.g., https://store.arduino.cc/catalogsearch/result/?q=mkr[MKR boards], https://store.arduino.cc/arduino-nano-33-iot[Nano 33 IoT], https://store.arduino.cc/arduino-zero[Zero]) to play back .wav files from a storage device like an SD card to the `DAC0`/`A0` pin.
```
I didn't realize this readme is asciidoc, not Markdown, so I used the wrong link syntax in my previous suggestion. Sorry about that!